script/policy_source_helper.py

```diff
@@ -87,6 +87,48 @@ def AddBravePolicies(template_file_contents):
             'desc': ('''This policy allows an admin to specify that Brave '''
                      '''Wallet feature will be disabled.'''),
         },
+        {
+            'name': 'ShieldsBlockedForUrls',
```

Comment:
Please use consistent terminology for shields (enabled/disabled).

Reply:
this applies everywhere

Reply:
block/allow is very confusing when applied to shields and should only be used internally to check content settings. Everything else should reference enable/disable

Reply:
@bridiver this is matching some of what Chromium has for policy names:
- https://chromeenterprise.google/policies/#ClipboardAllowedForUrls
- https://chromeenterprise.google/policies/#ClipboardBlockedForUrls
- https://chromeenterprise.google/policies/#CookiesAllowedForUrls
- etc

Reply:
@bsclifton that doesn't matter. Javascript blocked/allowed makes sense and is easy to understand. The same with cookies, etc... We need to use consistent terminology here and shields blocked/allowed is not consistent and it's extremely confusing.

Reply:
to be clear I'm not talking about removing `ForUrls`, just using enabled/disabled instead of allow/block. Honestly without looking at the code I couldn't even tell you whether allow is enabled or disabled and that's the problem with using allow/block with shields

Reply:
I can see how block/allow would be confusing for shields - maybe we can change these to 
- `ShieldsEnabledForUrls`
- `ShieldsDisabledForUrls`

Reply:
renamed